include/pybind11/detail/common.h

```diff
@@ -913,5 +913,15 @@ inline static std::shared_ptr<T> try_get_shared_from_this(std::enable_shared_fro
 #endif
 }
 
+#if defined(_MSC_VER) && _MSC_VER <= 1916
+// warning C4100: Unreferenced formal parameter
+template <typename... Args>
+inline constexpr void workaround_incorrect_msvc_c4100(Args &&...) {}
+#    define PYBIND11_WORKAROUND_INCORRECT_MSVC_C4100(...)                                         \
+        detail::workaround_incorrect_msvc_c4100(__VA_ARGS__)
```

Comment:
The structure here is very visually confusing to me. If this was on a single line it would be easier to read, or if it had a space above it, and/or if the define was not indented, or if the template was indented.

I think its largely due to the double-indent of the body but no indent on the define above.

Reply:
It's clang-format'ed.
I could make it different now, but later clang-format would change it straight back to this.
Do you want me to change it anyway?

Reply:
Adding a blank line or two won't be removed, and that would help.

Reply:
Please take another look. I added three, thinking that's most clear.
I also ran it through clang-format-diff again, which left it like what you see.